spec/views/post_index_spec.rb

```diff
@@ -23,24 +24,23 @@
     visit '/user/1/post'
   end
 
-  it 'displays the author name' do
-    expect(page).to have_content('John Doe')
-  end
-
-  it 'display total comments and likes for the post' do
-    expect(page).to have_content(post1.comments_counter)
-    expect(page).to have_content(post1.likes_counter)
+  it "I can see the user's name." do
+    visit '/user/1/post'
+    page.has_content?(user.name)
   end
 
-  it 'displays the first comment' do
-    expect(page).to have_content(post1.comments.first)
+  it 'I can see the number of posts the user has written.' do
+    visit '/user/1/post'
+    page.has_content?(user.posts_counter)
   end
 
-  it 'displays the total number of posts' do
-    expect(page).to have_content("Number of posts: #{user.posts_counter}")
+  it "I can see a post's title." do
+    visit '/user/1/post'
+    page.has_content?(post.title)
   end
 
   it 'has a link to view each post' do
     visit '/user/1/post'
+    page.has_content?(post.text)
   end
```

Comment:
- [ ] Let's ensure to include test for the below scenarios for _post index_ views: 
   * I can see the user's profile picture
   * I can see the first comments on a post
   * I can see how many comments a post has
   * I can see how many likes a post has
   * I can see a section for pagination if there are more posts than fit on the view
   * When I click on a post, it redirects me to that post's show page